extractor/filesystem/language/javascript/yarnlock/yarnlock.go

```diff
@@ -160,7 +160,7 @@ func determineYarnPackageResolution(props []string) string {
 }
 
 func parseYarnPackageGroup(desc *packageDescription) *extractor.Inventory {
-	name := extractYarnPackageName(desc.header)
+	name := extractYarnPackageName(strings.TrimSuffix(desc.header, ":"))
```

Comment:
this should be handled inside of `extractYarnPackageName` as that's the responsibility of that particular function